src/assets/i18n/fr.json5

```diff
@@ -5510,6 +5564,27 @@
   // "search.filters.search.submit": "Submit",
   "search.filters.search.submit": "Soumettre",
 
+  //"search.filters.operator.equals.text": "Equals",
+  "search.filters.operator.equals.text": "Est égal à",
+
+  //"search.filters.operator.notequals.text": "Not Equals",
+  search.filters.operator.notequals.text": "N'est pas égal à",
+
+  //"search.filters.operator.authority.text": "Authority",
+  "search.filters.operator.authority.text": "Est une autorité",
+
+  //"search.filters.operator.notauthority.text": "Not Authority",
+  "search.filters.operator.notauthority.text": "N'est pas une autorité"
```

Comment:
This line is missing a trailing comma (,)

Reply:
Done. Thanks @tdonohue for pointing me out the 2 errors.